bindings/matrix-sdk-crypto-js/src/verification.rs

```diff
@@ -906,6 +908,20 @@ impl VerificationRequest {
         self.inner.timed_out()
     }
 
+    /// The number of milliseconds remaining before this verification flow times
+    /// out.
+    ///
+    /// Returns zero if the time has already passed.
+    #[wasm_bindgen(js_name = "timeRemainingMillis")]
+    pub fn time_remaining_millis(&self) -> f64 {
```

Comment:
I feel like this belongs into the crypto crate. Otherwise we're running the risk of the other bindings re-implementing.

Reply:
done.